src/shared/calendarspec.c

```diff
@@ -872,6 +913,38 @@ static int parse_calendar_time(const char **p, CalendarSpec *c) {
         return 0;
 }
 
+static int parse_and_every(const char *s, usec_t *ts, usec_t *rep) {
+        _cleanup_free_ char *p_ts = NULL;
+        int i, r;
+
+        char const *p = strcasestr(s, " and ");
```

Comment:
Please specify `const` at first.

I think we should only accept ` and every ` in lower case.
Please use case sensitive ones here and below.